### PR TITLE
The environment variable should be BRANCH_NAME

### LINKED ISSUE #483: jenkins master build not pushing to quay
The last successful master build produced the following output message despite succeeding:
```
not master branch, not pushing to docker repo
```

### PR DESCRIPTION
**Pull request by joejulian**
According to
https://common-jenkins.kubeme.io/job/K2/pipeline-syntax/globals#env the
correct variable name is BRANCH_NAME instead of BRANCH

Fixes #483

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -96,7 +96,7 @@ podTemplate(label: 'k2', containers: [
             //only push from master.   assume we are on samsung-cnct fork
             //  ToDo:  check for correct fork
             stage('docker push') {
-                if (env.BRANCH == "master") {
+                if (env.BRANCH_NAME == "master") {
                     sh 'docker push quay.io/coffeepac/k2:jenkins'
```
Comment (DStorck) [inline Jenkinsfile]: why is this pushing to @coffeepac 's quay repo?

Comment (joejulian) [inline Jenkinsfile]: apparently he's taking ownership and now we can route any support questions to him. :rofl:

Comment (coffeepac) [inline Jenkinsfile]: can you update this to point to samsung_cnct instead of coffeepac?

File: Jenkinsfile [hunk 2]
```diff
@@ -96,8 +96,8 @@ podTemplate(label: 'k2', containers: [
             //only push from master.   assume we are on samsung-cnct fork
             //  ToDo:  check for correct fork
             stage('docker push') {
-                if (env.BRANCH == "master") {
-                    sh 'docker push quay.io/coffeepac/k2:jenkins'
+                if (env.BRANCH_NAME == "master") {
+                    sh 'docker push quay.io/samsung-cnct/k2:latest'
```
Comment (coffeepac) [inline Jenkinsfile]: need to change it in the build step a few lines up as well.

**Comment by coffeepac** [comment]
and please change 'coffeepac-quay-robot-dockercfg' to 'samsungcnct-quay-robot-dockercfg'

**Comment by DStorck** [comment]
LGTM

**Pull request merged**